src/main/resources/org/jenkinsci/plugins/github_branch_source/GitHubConfiguration/config.jelly

```diff
@@ -1,5 +1,10 @@
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
+    <f:section title="GitHub API usage">
+        <f:entry title="${%Check Github API rate limit mode}" field="apiRateLimitChecker">
```

Comment:
nit: I might reword this a bit, but I'd see what other reviewers think before changing anything.

```suggestion
        <f:entry title="${%Github API usage rate limiting strategy}" field="apiRateLimitChecker">
```

It might also be good to add a `help-apiRateLimitChecker.html` file with more details about what the field does for users.